samples/metrics_monitor/test/cttmetrics_gtest.cpp

```diff
@@ -31,7 +31,11 @@ File Name: cttmetrics_gtest.cpp
 #include "cttmetrics.h"
 #include "cttmetrics_utils.h"
 #include "gtest/gtest.h"
-#include "igt_load.h"
+
+extern "C" {
```

Comment:
It's better to add this into .h files directly:
```
#ifdef __cplusplus
extern "C" {
#endif 
```

Reply:
done

Reply:
no, not done. Why you did that here in .cpp? I talked about #include "device_info.h" and  #include "igt_load.h". Here in  .cpp you need to completely delete extern "C" and just include files directly.